app/User.php

```diff
@@ -128,6 +132,18 @@ public function roles()
         return $this->belongsToMany('App\Role');
     }
 
+    public function hasRole($roleName)
```

Comment:
tato metoda tu je zbytocna, prepisujes uz existujucu. Vid [https://github.com/Zizaco/entrust#checking-for-roles--permissions](https://github.com/Zizaco/entrust#checking-for-roles--permissions)